src/js/containers/router/RouterRoutes.jsx

```diff
@@ -317,7 +330,7 @@ const routeDefinitions = {
         },
         {
             path: 'detachedHelp',
-            onEnter: checkHelpUserPermissions,
+            onEnter: checkUserPermissions,
```

Comment:
It feels redundant to check permissions when entering the Help page. Really all we need to be doing is checking if they are logged in I think? Or do we not want them going to any of the DAIMS or practices or any of that if they have help only permissions? Because this check would always bounce them out of it to the DABS help page.